latexify/core.py

```diff
@@ -20,16 +20,9 @@
 
 import dill
 
-_MATH_SYMBOLS = {
-    'aleph', 'alpha', 'beta', 'beth', 'chi', 'daleth',
-    'delta', 'digamma', 'epsilon', 'eta', 'gamma', 'gimel',
-    'iota', 'kappa', 'lambda', 'mu', 'nu', 'omega', 'omega',
-    'phi', 'pi', 'psi', 'rho', 'sigma', 'tau', 'theta',
-    'upsilon', 'varepsilon', 'varkappa', 'varphi', 'varpi', 'varrho',
-    'varsigma', 'vartheta', 'xi', 'zeta', 'Delta', 'Gamma',
-    'Lambda', 'Omega', 'Phi', 'Pi', 'Sigma', 'Theta',
-    'Upsilon', 'Xi',
-}
+from latexify import predefined_variables
+
+_MATH_SYMBOLS = predefined_variables.get_math_symbols()
```

Comment:
It is better to remove `_MATH_SYMBOLS` and use the imported dictionary directly.

Reply:
Fixed in the following commits.